pkg/webhook/webhook.go

```diff
@@ -89,8 +82,6 @@ func NewPodMutator(client client.Client, reader client.Reader, arcCluster bool,
 
 // PodMutator adds projected service account volume for incoming pods if service account is annotated
 func (m *podMutator) Handle(ctx context.Context, req admission.Request) (response admission.Response) {
-	warnings := []string{}
```

Comment:
Can we remove the `syncBuffer` from the e2e test, I think that was forgotten in the last PR.